cv-data-controller/src/main/java/com/trihydro/cvdatacontroller/controller/ActiveTimHoldingController.java

```diff
@@ -211,14 +290,47 @@ public ResponseEntity<List<ActiveTimHolding>> getActiveTimHoldingForRsu(@PathVar
         } finally {
             try {
                 // close prepared statement
-                if (statement != null)
+                if (statement != null) {
                     statement.close();
+                }
                 // return connection back to pool
-                if (connection != null)
+                if (connection != null) {
                     connection.close();
+                }
                 // close result set
-                if (rs != null)
+                if (rs != null) {
                     rs.close();
+                }
+            } catch (SQLException e) {
+                e.printStackTrace();
+            }
```

Comment:
Suggestion (non-blocking): We should switch to using try-with-resources in this file

Reply:
Addressed